app/test/block_production_test.go

```diff
@@ -0,0 +1,52 @@
+package app
+
+import (
+	"testing"
+	"time"
+
+	appns "github.com/celestiaorg/celestia-app/pkg/namespace"
+	"github.com/celestiaorg/celestia-app/test/util/testnode"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/stretchr/testify/suite"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+)
+
+func TestBlockProductionestSuite(t *testing.T) {
+	suite.Run(t, new(BlockProductionestSuite))
+}
+
+type BlockProductionestSuite struct {
+	suite.Suite
+
+	accounts []string
+	cctx     testnode.Context
+}
+
+func (s *BlockProductionestSuite) SetupSuite() {
+	if testing.Short() {
+		s.T().Skip("skipping full node integration test in short mode.")
+	}
+	t := s.T()
+
+	accounts := make([]string, 40)
+	for i := 0; i < 40; i++ {
+		accounts[i] = tmrand.Str(10)
+	}
+
+	cfg := testnode.DefaultConfig().
+		WithAccounts(accounts).
+		WithTimeoutCommit(10 * time.Second)
+
+	cctx, _, _ := testnode.NewNetwork(t, cfg)
+	s.cctx = cctx
+	s.accounts = accounts
+}
+
+func (s *BlockProductionestSuite) Test_PostData() {
```

Comment:
[optional] can this test include a comment on what it it's attempting to verify or rename to something more descriptive? Two options b/c I'm not strongly opinionated

```go
func (s *BlockProductionestSuite) TestInabilityToIncludeTransactionsInBlockOne() {
```

```go
Test_PostData verifies that no transactions can be included in the block at height one.
func (s *BlockProductionestSuite) Test_PostData() {
```

Reply:
Good suggestion, thanks! please see https://github.com/celestiaorg/celestia-app/pull/2220/commits/17979e8aa4e38fa32100d2c77d06a5892fa42881